.gitmodules

```diff
@@ -16,3 +16,9 @@
 [submodule "deps/hat-trie"]
 	path = deps/hat-trie
 	url = https://github.com/Tessil/hat-trie.git
+[submodule "continuable"]
+	path = deps/continuable
+	url = https://github.com/Naios/continuable/
+[submodule "function2"]
+	path = deps/function2
+	url = https://github.com/Naios/function2
```

Comment:
it is a dependency for continuable

Reply:
Can't it be added automatically if we run `git submodule update --recursive` ?

Reply:
continuable adds too many deps (asio, boost, gtest), it is easier (and faster on CI) to clone only required dependencies